src/watchFixPlugin/index.js

```diff
@@ -0,0 +1,52 @@
+const chalk = require('chalk');
+const ora = require('ora');
+const configOverrides = require('../utils/configOverrides');
+
+const wait = time => new Promise(res => setTimeout(res, time));
+
+class ESLintWatchFixPlugin {
+  constructor({ stdout, config }) {
+    this._stdout = stdout;
+    this._key = config.key || 'F';
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async run() {
+    const fix = configOverrides.getFix();
+    configOverrides.setFix(!fix);
+    await this._showLoading(fix);
+    return true;
+  }
+
+  getUsageInfo() {
+    return {
+      key: this._key,
+      prompt: this._getPrompt(),
+    };
+  }
+
+  async _showLoading(fix) {
```

Comment:
why do we need this?

Reply:
It looks slick, but does it do anything? Other than delay for a tiny bit

Reply:
The only reason was to provide an indicator and feedback of what is happening, but I guess you are right... none of the other Jest keys does that, I just removed it.